plugins/modules/gitlab_label.py

```diff
@@ -0,0 +1,503 @@
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: gitlab_label
+short_description: Creates/updates/deletes GitLab Labels belonging to project or group.
+description:
+  - When a label does not exist, it will be created.
+  - When a label does exist, its value will be updated when the values are different.
+  - Labels can be purged
+  - Task can be run in check mode
+author:
+  - "Gabriele Pongelli (@gpongelli)"
+requirements:
+  - python >= 2.7
+  - python-gitlab python module
+extends_documentation_fragment:
+  - community.general.auth_basic
+  - community.general.gitlab
+  - community.general.attributes
+
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
+
+options:
+  state:
+    description:
+      - Create or delete project or group label.
+      - Possible values are present and absent.
+    default: present
+    type: str
+    choices: ["present", "absent"]
+  purge:
+    description:
+      - When set to V(true), delete all labels which are not mentioned in the task.
+    default: false
+    type: bool
+    required: false
+  project:
+    description:
+      - The path and name of the project, mandatory field but it's alternatively with group.
+    required: false
+    type: str
+  group:
+    description:
+      - The path of the group, mandatory field but it's alternatively with project.
+    required: false
+    type: str
+  labels:
+    version_added: 1.0.0
+    description:
+      - A list of dictionaries that represents gitlab project's or group's labels.
+    type: list
+    elements: dict
+    required: false
+    default: []
+    suboptions:
+      name:
+        description:
+          - The name of the label.
+        type: str
+        required: true
+      color:
+        description:
+          - The color of the label.
+          - Required when O(state=present).
+        type: str
+      priority:
+        description:
+          - Integer value to give priority to the label.
+        type: int
+        required: false
+        default: null
+      description:
+        description:
+          - Label's description
+        type: str
+        default: null
+      new_name:
+        description:
+          - Optional field to change label's name
+        type: str
+        default: null
+'''
+
+
+EXAMPLES = '''
+# same project's task can be executed for group
+- name: Create one Label
+  community.general.gitlab_label:
+    api_url: https://gitlab.com
+    api_token: secret_access_token
+    project: "group1/project1"
+    labels:
+      - name: label_one
+        color: #123456
+    state: present
+
+- name: Create many group labels
+  community.general.gitlab_label:
+    api_url: https://gitlab.com
+    api_token: secret_access_token
+    group: "group1"
+    labels:
+      - name: label_one
+        color: #123456
+        description: this is a label
+        priority: 20
+      - name: label_two
+        color: #554422
```

Comment:
```suggestion
        color: '#554422'
```
Same multiple times below.